deploy/vagrant/conf/ec2.yml

```diff
@@ -100,4 +100,14 @@ AMI: "ami-1ecae776"
 Security_Group: "tachyon-vagrant-test"
 
 Region: "us-east-1"
+
 Availability_Zone: "us-east-1a"
+
+# VPC and Subnet ID
+#
+# Depending on when you created your aws account, you may not have a default VPC and subnet set up for US-east-1.
```

Comment:
http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/default-vpc.html#default-vpc-basics
Some users do not have access to default VPC in all regions, and creating them involves contacting Amazon support.  This particular AMI would not launch under ec2-classic mode.